appendices.tex

```diff
@@ -370,6 +370,146 @@ \chapter{Listing of application roles}
 \item [yearend\_run]
 \end{description}
 
+\chapter{Deprecated Content}
+\label{app-deprecated-content}
+
+\section{ledgersmb.conf}
+\label{app-deprecated-ledgersmb-conf}
+
+This ledgersmb.conf is deprecated as of 1 Jan 2023 and only remains here for historical use.
+
+The environment variable for controlling the configuration is \texttt{LSMB\_CONFIG\_FILE}.
+
+By default it pointed to \texttt{/etc/ledgersmb/ledgersmb.conf} and can be set to an alternate location.
+
+\subsubsection{[main] Section}
+\label{subsubsec-global-config-ledgersmb-conf-general}
+
+\begin{description}[style=nextline]
+	\item[language] Set language for login and admin pages. 
+	\item[date\_format] Set the date format for the admin pages.  The default is \texttt{yyyy-mm-dd}. Unset means detected from the browser)
+	\item[log\_level] Specifies the logging level to be used. This setting is ignored when the \texttt{log\_config} setting 
+	is supplied and indicates an existing file. Available values are OFF, FATAL, ERROR, WARN, INFO, DEBUG, TRACE.
+	\item [log\_config] Set the name of the extended logging configuration file. This file uses the \texttt{log4j/Log::Log4perl} syntax described at \url{https://metacpan.org/pod/Log::Log4perl#Configuration-files}
+	\item [dojo\_theme] This is the Dojo theme that is used by default when no other theme
+	has been selected.  The default if unset is \texttt{claro}. Other values can be \texttt{nihilo}, \texttt{soria}, and \texttt{tundra}.
+	
+	
+% From the old book, commented out for now.
+% 	\item[auth] @@@TODO

```

Comment:
Indicates which authentication module is to be used; default is DB, referring to LedgerSMB::Auth::DB. However, in 1.10, this is configured through the "database factory dependency injection", which does take `host`, `port`, `sslmode` and `db_namespace` into account, but not the `auth` setting.